src/plugins/search-disabled-repos.py

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2015  Red Hat, Inc.
+#
+# Authors: Valentina Mukhamedzhanova <vmukhame@redhat.com>
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# the GNU General Public License v.2, or (at your option) any later version.
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY expressed or implied, including the implied warranties of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.  You should have received a copy of the
+# GNU General Public License along with this program; if not, write to the
+# Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+# 02110-1301, USA.  Any Red Hat trademarks that are incorporated in the
+# source code or documentation are not subject to the GNU General Public
+# License and may only be used or replicated with the express permission of
+# Red Hat, Inc.
+
+import logging
+import os
+
+from yum.plugins import TYPE_CORE, TYPE_INTERACTIVE
+from yum.constants import TS_INSTALL_STATES
+
+requires_api_version = '2.7'
```

Comment:
That's correct, to make this functionality possible we are going to update the API in RHEL7.2. I posted the link to the scratch build of yum here https://bugzilla.redhat.com/show_bug.cgi?id=1232232#c17